### PR TITLE
docs: update example in README

### PR DESCRIPTION
**Pull request by nfischer**
If you have any formatting issues with the example in the README, or want the example changed, feel free to modify as you see fit.

I haven't tested this example, but I'm pretty sure it's correct. Feel free to test it out.

### CODE REVIEW

**Comment by levithomason** [comment]
Looks good, I'll test it soon as I can.  While we're at cleanup, would you mind aligning the comments as well?  I think it helps with scanning readability.

```shell
$ shx pwd                        # ShellJS commands are supported automatically
/home/username/path/to/dir

$ shx ls                         # files are outputted one per line
file.txt
file2.txt

$ shx rm *.txt                   # a cross-platform way to delete files!

$ shx ls

$ shx echo "Hi there!"
Hi there!

$ shx touch helloworld.txt

$ shx cp helloworld.txt foobar.txt

$ shx mkdir sub

$ shx ls
foobar.txt
helloworld.txt
sub

$ shx rm -r sub                  # options work as well
```

**Comment by nfischer** [comment]
Good suggestion, will do

**Comment by nfischer** [comment]
@levithomason Updated with your suggestion

**Comment by levithomason** [comment]
Haven't tested but it looks like it should work as described.  LGTM

**Comment by nfischer** [comment]
Alright, I'll merge this in. If there's any issues, we can revise that in another PR.

File: README.md [hunk 1]
```diff
@@ -1,55 +1,62 @@
...
+$ shx pwd                       # ShellJS commands are supported automatically
 /home/username/path/to/dir
+
+$ shx ls                        # files are outputted one per line
+file.txt
+file2.txt
+
+$ shx rm *.txt                  # a cross-platform way to delete files!
+
 $ shx ls
```
Comment (ariporad) [inline README.md]: Shoudl we delete this too?

Comment (nfischer) [inline README.md]: I left it to demonstrate that files are deleted, as expected.

Comment (ariporad) [inline README.md]: Ok. You might want to add a comment describing that.

**Pull request merged**